server/controllers/authorController.js

```diff
@@ -24,6 +24,21 @@ class AuthorController {
       util.errorStatus(res, 500, error.name);
     }
   }
+
+  static async listAuthor(req, res) {
+    try {
+      if (!req.headers.authorization) {
```

Comment:
This can be a reusable middleware as it might be required in other routes. 
Also what if the authorization is invalid?